src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/__tests__/FlagConfigurationStepWidget.test.tsx

```diff
@@ -43,9 +41,18 @@ jest.mock('@pipeline/components/AbstractSteps/Step', () => {
   }
 })
 
-const renderComponent = (props: Partial<FlagConfigurationStepWidgetProps> = {}): RenderResult =>
+const renderComponent = (props?: Partial<FlagConfigurationStepWidgetProps>): RenderResult =>
```

Comment:
No need to make it optional, by defaulting to an empty object, it does that for you.
```suggestion
const renderComponent = (props: Partial<FlagConfigurationStepWidgetProps> = {}): RenderResult =>
```